libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -33,8 +38,26 @@ export class BookSearchComponent {
 
   addBookToReadingList(book: Book) {
     this.store.dispatch(addToReadingList({ book }));
-  }
 
+    this.store
+      .select(confirmedAddToReadingList)
```

Comment:
Handling actions belongs in the effects class. Either move the snackbar logic to an effect, or do `this.snackbar.open(...)` without listening on the action.